build.gradle.kts

```diff
@@ -193,14 +191,10 @@ allprojects {
     version = releaseVersion
     repositories {
         mavenCentral()
-        maven {
-            name = "buf"
-            url = uri("https://buf.build/gen/maven")
-        }
```

Comment:
Originally this project used a generated SDK, but it now builds the conformance protos as part of this project so we can remove this.

Reply:
👍. I really _wanted_ to use a generated SDK for the newer conformance protos, but then we don't have the option to build for both lite and standard Java runtimes. Do we? If there's a way to pick which runtime to use in the URL, I'm all ears and will happily move this back to using generated SDKs (since that's what we use in conformance tests for other languages that generated SDKs support).

Reply:
Yep - you can use a `lite` runtime artifact for plugins that support it: https://buf.build/docs/bsr/generated-sdks/maven#remote